libdevice/imf_wrapper.cpp

```diff
@@ -0,0 +1,162 @@
+//==----- imf_wrapper.cpp - wrappers for intel math library functions ------==//
```

Comment:
Maybe it would be more clear to name this file as imf_wrapper_fp32.cpp. And move all float16 related stuff to another file imf_wrapper_fp16.cpp.

Reply:
Hi, @akolesov-intel 
I agree that splitting all wrapper functions into fp32 and fp16 should be clear in source code and there is no difference in functionality. However, as we can see, the cmake for imf build is very complicated now, the reason is original cmake has some pre-assumption that all functions in the same wrapper/fallback devicelib are in one single source. I have some vague idea to simplify the cmake for imf devicelib and support source split in a clean way but haven't tried. Can we keep fp32 and fp16 in one source currently and split the files when the cmake refactor is done?